io/genbank/genbank.go

```diff
@@ -213,19 +211,12 @@ func WriteMulti(sequences []Genbank, path string) error {
 // Build builds a GBK byte slice to be written out to db or file.
 func Build(gbk Genbank) ([]byte, error) {
 	gbkSlice := []Genbank{gbk}
-	multiGBK, err := buildMultiNth(gbkSlice, -1)
+	multiGBK, err := BuildMulti(gbkSlice)
```

Comment:
@Koeng101 wasn't `buildMultiNth` meant to deal with some annoying corner case?

Reply:
Hmmm not entirely sure, I don't quite remember

Reply:
The build functions will mostly be moved into `bio` once we get there though, I just haven't started on genbank compatibility because it's gonna be quite sizable and deserves its own PR

Reply:
The underlying logic is the same, it was just being buried a few function calls deep for some reason.

Before: `Build()` makes a single element slice and passes it to `BuildMulti()`, which immediately calls `buildMultiNth`. The `count` argument was unused, that's why I deleted it.